results/utils.py

```diff
@@ -96,9 +96,12 @@ def get_npcp_data(
     
     if region == 'AU':
         shape = get_aus_shape()
-    else:
+        da = clip_data(ds[variable], shape)
+    elif region:
         shape = get_nrm_super_cluster(region)
-    da = clip_data(ds[variable], shape)
+        da = clip_data(ds[variable], shape)
+    else:
+        da = ds[variable]
                    
```

Comment:
@AliciaTak - We were clipping the data twice (once when we first got the data and again for the plotting) so I've edited the `get_npcp_data` function so the default is to not clip at all. That should help things run a little more efficiently.